src/styles/blocks/main-title.scss

```diff
@@ -0,0 +1,218 @@
+.main {
+
+  &__container {
+    padding: 0 20px;
+    margin-bottom: 60px;
+
+    @include onTablet {
+      padding: 0 35px;
+      margin-bottom: 100px;
+    }
+
+    @include onDesktop {
+      padding: 0 55px;
+      margin-bottom: 100px;
+    }
+
+    &--nevs {
+      display: block;
+      @include onDesktop {
+        display: grid;
+      }
+    }
+
+    &--date {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+
+      @include onTablet {
+        justify-content: flex-start;
+      }
+
+      @include onDesktop {
+        justify-content: flex-start;
+      }
+    }
+  }
+
+  &__exhib {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
```

Comment:
Font family, font style, and font weight are repetitive in multiple elements. Consider creating a mixin for font styles to avoid code repetition.